test/lib/generation/generator_test.dart

```diff
@@ -95,7 +110,8 @@ void main() {
       expect(result is String, true);
 
       // Do not modify
```

Comment:
We probably should remove this comment 😂

Reply:
On it